recipes/huffman/all/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+﻿cmake_minimum_required (VERSION 3.4)
```

Comment:
Please, send this cmake file (without conan_basic_setup stuff) to the upstream. I see no Cmake support there and no one tried to submit it.

Reply:
Here is the [pull request](https://github.com/drichardson/huffman/pull/7)